### ISSUE TITLE
EOY Social Creative: Round 2

### ISSUE THREAD

**Issue by tlasadea** [MozillaFoundation/Advocacy#1107]
This ticket is for the creation of additional creative to support the upcoming EOY fundraising campaign.

As mentioned in the kick-off call, we need:

_Two sets_ of creative for _two audiences_ (Mozilla Supporters & Conscious Choosers) _**across three mozilla-issues: Misinformation, Digital Inclusion and Net Neutrality/Decentralization.**_

In execution, this looks like:
**6 total creative executions for Misinformation (3 for Moz Supporters, 3 for CC) 
6 total creative executions for Digital Inclusion (3 for Moz Supporters, 3 for CC).**

**UPDATE: @kristinashu we're no longer doing decentralization, test shows it performed poorly (worse than control) and it's quite a thorny issue.** 

We'll also have some flights live with more granular timing that syncs up with the fundraising email calendar, as well as flight for the fundraising video Brett created.

In terms of campaign flighting, we won't be using sequences to keep the complexity of creative down, and all creative copy will have various degrees of donation asks - so effectively consideration + conversion flights.

Finally, this means we need *18 total lines of copy* CC @jonmozilla 

The brief for this work is [here](https://docs.google.com/document/d/1MHbq4sYAaKCoi51r5SZGUVrUoa5301xNCPg6VPpUrpY/edit).

CC @HPaulJohnson @kristinashu

**Comment by tlasadea** [comment]
@caltrider @jonmozilla lets have a meeting on Weds about copy for this? I'll schedule it.

**Comment by caltrider** [comment]
Hey @tlasadea, can we meet tomorrow?  I'm out on Wednesday.

**Comment by tlasadea** [comment]
​@caltrider yes!

**Comment by HPaulJohnson** [comment]
As background, here are the winning snippets from the last month with the "future of the Internet" messaging: 

_Will the future of the internet be a set of walled, private compounds with locked gates and restricted entry? Or an open, productive commons, where creativity and innovation flourish? As a not-for-profit, Mozilla’s core mission is to fight every day for a better and healthier internet, for all. We depend on contributions from people like you to support this mission, so please donate today._

_The future of the internet is at stake, with new threats to our online safety, privacy and security appearing almost every day. But like a doctor keeping tabs on a patient, Mozilla fights every day to protect a healthy internet, through grassroots advocacy work and software that enables the open web. As a not-for-profit we rely on your donations, so please contribute today._

There's also this helpful summary of all the winning tests from other channels so far this year: https://docs.google.com/document/d/199TDARstwLLvI-ljBdogQavvBojNGTJ_wHq-YWlawGg/edit

**Comment by HPaulJohnson** [comment]
@tlasadea @kristinashu per our conversation this a.m., here's a more detailed roundup of the creative results from the messaging test: 

https://docs.google.com/spreadsheets/d/12dP5_jZiBKQOc6MUUIdkNQXXnq2wSD5imv1YBnqwJL4/edit#gid=0

TL;DR Inclusion and Misinformation performed pretty consistently better than the control, while the Decentralization creative performed worse than the control in each phase. For decentralization, we should do a fresh take on this issue and retry, or drop it from the rotation. 

In terms of messaging, it seems like the "awareness" CTR were our highest so we should look to that copy as a baseline.

**Comment by jonmozilla** [comment]
My 2 cents: let's drop decentralization now so we can focus our resources
on what's working (I don't think we have the time).

**Comment by tlasadea** [comment]
Thanks, so @kristinashu I have updated the brief, you should have enough info to get started now.

@caltrider @jonmozilla I have put a meeting in to brief re copy.

**Comment by kristinashu** [comment]
Thanks for the brief. Here's a summary of the deliverables I'll be working on:
- 3 Misinformation executions for Mozilla Supporters (bold but slightly reduced controversy)
- 3 Misinformation executions for Conscious Choosers (bold & similar to the testing creative)
- 3 Digital Inclusion executions for Mozilla Supporters (bold but slightly reduced controversy)
- 3 Digital Inclusion executions for Conscious Choosers (bold & similar to the testing creative)

Social share images:
- 2 generic share images for when people have donated and they want to share on Facebook or Twitter (bolder creative approach)

**Comment by kristinashu** [comment]
@tlasadea @HPaulJohnson here are some iterations https://redpen.io/p/zj3a03844f39d2841c

I think we should have few different headlines and text messages to cycle through/text. Temi, if you write them I can quickly update the images.

For the generic social share images, does the campaign have a tagline or theme I should use?

**Comment by tlasadea** [comment]
@ScottDowne @alanmoo hey you two, @kristinashu is working on updating the share images for the donate LP - will pop them here when they're done.

**Comment by alanmoo** [comment]
Sorry, I don't understand @tlasadea - What will need to be done with these? (And what's "LP" stand for here?)

**Comment by tlasadea** [comment]
@alanmoo oh sorry. the donate landing page/site - you can share via social on FB and Twitter when you've donated. Kristina is updating the image that is shared when that happens. A la: https://twitter.com/jedidiahpark/status/935564682684923905

**Comment by alanmoo** [comment]
Ah, the OpenGraph images. Good, that needs an update 👍

**Comment by tlasadea** [comment]
@kristinashu @HPaulJohnson 

Reiterating our call this morning:


1) Update the Misinformation mobile phone creative to the "President Says Canada Should Never Had Independence" image
2) Export Misinformation (no author with 'Fake' stamp version) for Facebook image size
3) Develop another execution for Misinformation
4) Update digital inclusion blue highlighter image with new copy in RedPen
5) Export inclusion text message versions for Facebook image sizes 
6) Develop another execution for digital inclusion based on the ideas we talked about in the call (mirror/reflection of world vs tech bro internet & riffing off UN female sexism ads)

We'll have a check-in on Friday, and then another one on Tuesday Dec 5th. 

Also as promised, here are some examples of the propoganda/ Trump magazine covers that I was talking bout (to help with no 3).

![1503109075852](https://user-images.githubusercontent.com/30296602/33390108-dc7a82c2-d52c-11e7-9727-3780838432ab.jpg)
![star-wars-propaganda-poster-set_page_10](https://user-images.githubusercontent.com/30296602/33390114-e07006cc-d52c-11e7-9aa8-0005605a49a5.jpg)
![600fcf92e3dc432377fb218201fab8a9--social-network-internet](https://user-images.githubusercontent.com/30296602/33390119-e62f4ac8-d52c-11e7-83e9-0147fbdb2c31.jpg)
![trump-magazine-covers-der-speigel-feb-4-2017-jpeg](https://user-images.githubusercontent.com/30296602/33390124-e9a34498-d52c-11e7-85b2-9c574e157c45.jpeg)
![super-mario-propaganda-posters-16](https://user-images.githubusercontent.com/30296602/33390126-eb70c2aa-d52c-11e7-8fe5-f9db7ecf7454.jpg)

**Comment by kristinashu** [comment]
I've updated [redpen](https://redpen.io/p/zj3a03844f39d2841c) with the selected images and put placeholders for the new directions I will be working on.

Here are the generic share images for FB (1200 x 630 px) and Twitter (1200 x 601 px): https://drive.google.com/open?id=166r2cB5jzmdLd1qB1ikDJ4vCX3CWpGnd

@HPaulJohnson @tlasadea let me know if I've missed anything. @natalieworth let me know if you have any feedback or other ideas!

**Comment by tlasadea** [comment]
@alanmoo [here](https://drive.google.com/drive/folders/166r2cB5jzmdLd1qB1ikDJ4vCX3CWpGnd) are the images for OpenGraph.

**Comment by alanmoo** [comment]
Great. Can you file a new ticket with that link and the actual request for where to put them? This ticket's become very hard to follow and I don't want anyone jumping in to have to try and parse it all.

Update: #1108 (though #1110 was filed too)

**Comment by kristinashu** [comment]
@tlasadea updated creative is here! https://redpen.io/p/gm4e1ce11dc9e45341

**Comment by tlasadea** [comment]
Final creative for this is [here](https://drive.google.com/open?id=1h05GkySoOe9sICUy-MDMLHN5I4N2dQIV), and this has been trafficked for Alex, so closing this CC @HPaulJohnson.
